crates/orderbook/src/order_quoting.rs

```diff
@@ -713,7 +745,8 @@ mod tests {
                     sell_token_price: 0.2,
                 },
                 kind: OrderKind::Sell,
-                expiration: now + chrono::Duration::seconds(QUOTE_VALIDITY_SECONDS),
+                expiration: now + Duration::seconds(STANDARD_QUOTE_VALIDITY_SECONDS),
```

Comment:
Unrelated to this PR (so not asking it to be done here) - but we should probably give the same "allow for configuration" for the `STANDARD_QUOTE_VALIDITY_SECONDS` parameter.